target/zephyr/Kconfig

```diff
@@ -86,6 +86,13 @@ if MENDER_MCU_CLIENT
             help
                 Interval used to periodically check for new deployments on the Mender server.
 
+        config MENDER_DEPLOYMENT_LOGS
+            bool "Deployment logs"
+            default y
+            select FCB
+            help
+                Enable the deployment logs feature.
```

Comment:
I would extend here a bit more the help text. For example:

This is a feature that saves in non-volatile storage the Mender MCU warning and error logs during a deployment so that they can be send to the Mender Server in case on a failed deployment for further diagnostics. It incurs flash wear.